keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -2312,6 +2312,8 @@ in back-to-back releases.
   If multiple admission policies require the same conversion, convert only once. 
   From @liggitt: "webhook code loops up one level, first accumulates all the validation webhooks we'll run, then converts to the versions needed by those webhooks then evaluates in parallel"
 - authz check to the specific resource referenced in the policy's paramKind. ([comment](https://github.com/kubernetes/kubernetes/pull/113314#discussion_r1013135860))
+- complete feature of access to namespace metadata 
+- complete type check for CRD
```

Comment:
Since we're going to beta, how are we handling backward compatibility?

Reply:
Thank you for adding the rollout strategy! It is a great way to support adding new stuff.
Beside that, during offline discussion, we don't really care backward compatibility with Alpha feature much since the general suggested practice is to not rollback to a cluster which has alpha feature turned on and we allow breaking changes in alpha phrase. Thank you

Reply:
SGTM